src/Fable.Core/Fable.Core.Types.fs

```diff
@@ -240,3 +240,11 @@ type [<Erase>] U9<'a, 'b, 'c, 'd, 'e, 'f, 'g, 'h, 'i> =
     static member op_ErasedCast(x:'i) = Case9 x
     static member inline op_ErasedCast(x:'t) : U9<_, _, _, _, _, _, _, _, ^U> =
         Case9 (^U: (static member op_ErasedCast: 't -> ^U) x)
+
+
+type PtrType =
```

Comment:
Move to Fable.Core.Rust

Reply:
@alexswan10k Looks great, very minimal.

Very minor suggestion, do we want to rename `PointerType`/`PtrType` to `ReferenceType`/`RefType`, just to keep it aligned with .NET lingo? Also, we may need the name for `PointerAttribute` (or `PtrAttribute`) to represent actual non-rc pointers (for webasm/js interop etc.).